iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/BusbarSectionPosition.java

```diff
@@ -21,10 +21,16 @@ default String getName() {
         return NAME;
     }
 
+    /**
+     * The position of the busbar relative to other busbars.
+     */
     int getBusbarIndex();
 
     BusbarSectionPosition setBusbarIndex(int busbarIndex);
 
+    /**
+     * The position of the section relative to the other section of the busbar.
```

Comment:
What is a section here ? We only have `busBarSection` objects in IIDM. Same remark as before.

Reply:
as it is described it is a section of a busbar (busbarSection)